src/Middleware/Diagnostics/src/ExceptionHandler/ExceptionHandlerFeature.cs

```diff
@@ -1,14 +1,16 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable enable
+
 using System;
 
 namespace Microsoft.AspNetCore.Diagnostics
 {
     public class ExceptionHandlerFeature : IExceptionHandlerPathFeature
     {
-        public Exception Error { get; set; }
+        public Exception Error { get; set; } = default!;
```

Comment:
💡 Use `null` instead of `default` for reference types

```suggestion
        public Exception Error { get; set; } = null!;
```

Reply:
We've been using `default!` all over the place. Is that a preference or is there to more to using `null`?

Reply:
`default` suggests that the type is a value type or an unconstrained generic. `null` suggests that the type is a reference type or `Nullable<T>`.

Reply:
It'd be inconsistent to have it use `default!` in some places but not the other. I'll leave it to someone to bulk update the repo to use `null!` exclusively.